frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts

```diff
@@ -0,0 +1,207 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { BaseComponent } from '@core/base/base.component';
+import { getDisplayDate } from '@core/helpers/date-helper';
+import { getDisplayDuration } from '@core/helpers/display-duration-hepler';
+import { IDuration } from '@core/models/IDuration';
+import { INewMeeting } from '@core/models/INewMeeting';
+import { INewMeetingMember } from '@core/models/INewMeetingTeamMember';
+import { NewMeetingService } from '@core/services/new-meeting.service';
+import { NotificationService } from '@core/services/notification.service';
+import { naturalNumberRegex, newMeetingNameRegex } from '@shared/constants/model-validation';
+import { LocationType } from '@shared/enums/locationType';
+import { UnitOfTime } from '@shared/enums/unitOfTime';
+import { map, Observable, startWith } from 'rxjs';
+
+@Component({
+    selector: 'app-new-meeting',
+    templateUrl: './new-meeting.component.html',
+    styleUrls: ['./new-meeting.component.sass'],
+})
+export class NewMeetingComponent extends BaseComponent implements OnInit {
+    constructor(private newMeetingService: NewMeetingService, public notificationService: NotificationService) {
+        super();
+        this.getTeamMembersOfCurrentUser();
```

Comment:
Why in the constructor?

Reply:
Remove calling this method from constructor.